src/bind.js

```diff
@@ -18,8 +18,10 @@
  *
  * @return {Function}
  */
-function bind(callback) {
-  // write code here
+function bind(callback, ...otherParams) {
+  return (...moreParams) => {
```

Comment:
write it in one line
```suggestion
  return (...moreParams) => callback(...otherParams, ...moreParams);
```